bin/proving-service/Cargo.toml

```diff
@@ -24,7 +24,8 @@ async-trait = "0.1"
 axum = { version = "0.7" }
 bytes = "1.0"
 clap = { version = "4.5", features = ["derive", "env"] }
-figment = { version = "0.10", features = ["toml", "env"] }
+dotenvy = "0.15"
+figment = { version = "0.10", features = ["env"] }
```

Comment:
Do we still need `figment`?

Also, given the discussion in https://github.com/0xPolygonMiden/miden-node/discussions/650#discussioncomment-12051185, maybe we don't need the `dotenvy` dependency either?